CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,45 @@
+# grammY code of conduct
+
+## Who We Want to Be
+
+We as a community want to be inclusive for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, language, personal appearance, religion, or sexual identity and orientation.
+
+We are humans.
+We meet each other with respect and dignity.
+
+If it is not obvious what this means, here are a few examples of behavior that we do not tolerate.
+
+- Trolling, insulting or derogatory comments, and personal or political attacks
+- Public or private harassment
+- Publishing others' private information, such as a physical or email
+  address, without their explicit permission
+
+## Enforcement
+
+Admins enforce our standards of acceptable behavior.
+They take appropriate and fair corrective action in response to any behavior that they deem inappropriate, threatening, offensive, or harmful.
+When they take action, they will communicate this appropriately.
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to the admins at <https://t.me/grammyjs>.
+In the Telegram group chat, you can tag all available admins using the `/report` command.
+All complaints will be reviewed and investigated promptly and fairly.
+
+Admins respect the privacy and security of the reporter of any incident.
+
+### Guidelines
+
+Admins will try to follow these guidelines when taking action.
+
+- Above all, use human judgement which is appropriate for the situation.
+- Warn or temp ban people who use inappropriate language or show other unwelcome behavior.
+  If there is hope that the respective person can learn and avoid such behavior in the future, it is imperative to teach them rather than banning them.
+- Be transparent about your actions, and provide necessary clarity why you performed them.
+- Delete messages that are too harmful to stay online and be read by others.
+  Only delete these messages, and no others, so that the context of your actions is preserved as much as possible.
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant](https://www.contributor-covenant.org), version 2.0, available at <https://www.contributor-covenant.org/version/2/0/code_of_conduct.html>.
```

Comment:
```suggestion
This Code of Conduct is adapted from [Contributor Covenant](https://www.contributor-covenant.org), version 2.0, available at <https://www.contributor-covenant.org/version/2/0/code_of_conduct.html>.
```

Reply:
Note also that this was written by a group of native speakers, not me